src/server/system_services/bucket_server.js

```diff
@@ -468,6 +475,9 @@ async function read_bucket_sdk_info(req) {
             ),
             read_resources: _.map(bucket.namespace.read_resources, rs => pool_server.get_namespace_resource_extended_info(rs))
         };
+        if (bucket.namespace.caching) {
+            reply.namespace.caching = bucket.namespace.caching;
```

Comment:
No need to separate the new property from the others:
```suggestion
        reply.namespace = {
            write_resource: pool_server.get_namespace_resource_extended_info(
                system.namespace_resources_by_name[bucket.namespace.write_resource.name]
            ),
            read_resources: _.map(bucket.namespace.read_resources, rs => pool_server.get_namespace_resource_extended_info(rs))
            caching: bucket.namespace.caching,
        };
```

Reply:
ok